In the controller class, added the getAllEntries method (lines 200 - …

slymbugua opened the pull request:
…212).

The method does not really retrieve entries from a log file. It appends every log onto a text area which may not always be visible. The method is synchronized and is not complete yet while it awaits the completion of other methods and classes.

In the connect class, added three methods for querying tables (lines 123 - 171). The three methods take input from the UI such as ship size and resource type and queries tables to be used in the UI.

Commit:
In the controller class, added the getAllEntries method (lines 200 - 212). 
The method does not really retrieve entries from a log file. It appends every log onto a text area which may not always be visible. The method is synchronized and is not complete yet while it awaits the completion of other methods and classes.

In the connect class, added three methods for querying tables (lines 123 - 171). The three methods take input from the UI such as ship size and resource type and queries tables to be used in the UI.

tom-helfrich reviewed (changes requested):
getShip() method: remove "speed parameter"; FuelCapacity should be fuelCapacity, CargoCapacity should be cargoCapacity;
getPlanet() method: remove "isColonized";
remove getPerson() method, replace with getIdentifier()- (parameters in Identifier table that you would want to pull are "type" and "value")

Commit:
In the controller class, added the getAllEntries method (lines 200 - 212). 
The method does not really retrieve entries from a log file. It appends every log onto a text area which may not always be visible. The method is synchronized and is not complete yet while it awaits the completion of other methods and classes.

In the connect class, added three methods for querying tables (lines 123 - 171). The three methods take input from the UI such as ship size and resource type and queries tables to be used in the UI.

Removed speed parameter from getShip and isColonized from getPlanet methods. 
Changed getPerson method to getIdentifier method
